ci/docker/docker_build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Builds a docker image nighthawk-dev:latest containing the stripped binaries
+# based on a pre-build bazel-bin directory (with "-c opt" set).
+
+# Stop on errors.
+set -e
+
+# NOTE: explicit no -x for verbose commands. Because this is run in CI, doing so may result in
+# publishing sensitive information into public CI logs if someone makes a change in a
+# consuming script that is off guard.
+
+DOCKER_NAME="nighthawk"
+DOCKER_IMAGE_PREFIX="envoyproxy/${DOCKER_NAME}"
+BINARIES=(nighthawk_test_server nighthawk_client nighthawk_service nighthawk_output_transform)
+BAZEL_BIN=$(bazel info -c opt bazel-bin)
+WORKSPACE=$(bazel info workspace)
+TMP_DIR=${WORKSPACE}/tmp-docker-build-context
```

Comment:
Nit: please `""` wrap all the expansions in this file for whitespace safety.

Reply:
Done in 334fa1281e43f78b5b4a1cec6052d7ab98a5423e